hidden/Common/Util.swift

```diff
@@ -140,6 +140,19 @@ class Util {
     }
     
     
+    static func getEnablePermHide() -> Bool {
+        return UserDefaults.standard.bool(forKey: IS_PERM_HIDE_ENABLED)
+    }
+    
+    static func setEnablePermHide(_ isPermHideEnabled: Bool) {
+        UserDefaults.standard.set(isPermHideEnabled, forKey: IS_PERM_HIDE_ENABLED)
+    }
```

Comment:
Use computer Property for simple use.
static var isPermEnable: Bool {
get{}
set{}
}

Reply:
Should I change the rest of the Preference Items as well?

Reply:
No need, I will do it later

Reply:
I've changed this in the latest commit. Also changed the rest of the Preference Items.